src/SignalR/common/SignalR.Common/src/Microsoft.AspNetCore.SignalR.Common.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <Description>Common serialiation primitives for SignalR Clients Servers</Description>
     <TargetFrameworks>netstandard2.0;netcoreapp3.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(DotNetBuildFromSource)' == 'true'">netcoreapp3.0</TargetFrameworks>
```

Comment:
Is it possible to make this a more generic thing in https://github.com/aspnet/AspNetCore/tree/master/eng/targets so we don't pollute all our .csproj's?

Reply:
Yea maybe probably